.travis.yml

```diff
@@ -0,0 +1,19 @@
+# Based on the example travis in hererocks
+language: python
+sudo: false
+
+env:
+  - LUA="lua 5.3"
+
+before_install:
```

Comment:
we can install `love2d` as well and just run `love src` in the script to see if running it passes

Reply:
I'll add love src in the other PR.